src/app/cli/src/graphql.ml

```diff
@@ -1293,6 +1308,26 @@ module Queries = struct
               Public_key.Compressed.Set.mem propose_public_keys pk
           ; path= Secrets.Wallets.get_path (Coda_lib.wallets coda) pk } )
 
+  let transaction_status =
+    result_field "transactionStatus" ~doc:"Get the status of a transaction"
+      ~typ:(non_null Types.transaction_status)
+      ~args:
+        Arg.
+          [ arg "payment" ~typ:(non_null string)
+              ~doc:(Doc.bin_prot "Serialized payment") ]
```

Comment:
Can we just say "UserCommand id" without the bin_prot bit and make the type `non_null guid`?
I wonder if the arg should be called "id" or "userCommand" instead.
Then I think we're good!

Reply:
Right now, its not an id. We are actually passing in the bin_prot of a user_command. I can just call it `userCommand`.